client/src/Components/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
name components in better way , which gives a idea about what component is doing , for all components